corehq/apps/domain/models.py

```diff
@@ -309,6 +309,7 @@ class Domain(QuickCachedDocumentMixin, BlobMixin, Document, SnapshotMixin):
     is_active = BooleanProperty()
     date_created = DateTimeProperty()
     default_timezone = StringProperty(default=getattr(settings, "TIME_ZONE", "UTC"))
+    default_geocoder_location = StringProperty()
```

Comment:
There's a `DictProperty` you can use instead of storing this as a string, which would make it more semantic.

Reply:
changed to DictProperty